openaerostruct/docs/advanced_features/aerostruct_ffd.rst

```diff
@@ -9,11 +9,5 @@ This allows for more general design shape changes and helps sync up geometry cha
 .. warning::
   This example requires `pyGeo`, an external code developed by the MDO Lab. You can install it from `here <https://github.com/mdolab/pygeo>`_.
 
-.. literalinclude:: /../tests/test_aerostruct_ffd.py
-  :start-after: checkpoint 0
-  :end-before: checkpoint 1
-  :dedent: 8
-
-.. code-block:: python
-
-  print(prob["AS_point_0.fuelburn"][0])
+.. embed-code::
+  openaerostruct.tests.test_aerostruct_ffd.Test.test
```

Comment:
This isn't a comment directly on the changes in this PR, but do we need all the commented out code in the test?

Reply:
I was wondering the same thing...I noticed a lot of code commented out to use SNOPT, but we only use the scipy drivers in the tests.

Reply:
@kanekosh?

Reply:
I don’t think we need the commented out code.

Reply:
I just went through and removed the commented out code from the tests.